.circleci/config.yml

```diff
@@ -474,7 +474,7 @@ jobs:
                 command: LIBP2P_NIXLESS=1 GO=/usr/lib/go/bin/go make libp2p_helper
             - run:
                   name: Run unit tests
-                  command: ./scripts/skip_if_only_frontend_or_rfcs.sh bash -c 'source ~/.profile && export make build && (dune runtest src/lib --profile=dev -j8 || (./scripts/link-coredumps.sh && false))'
```

Comment:
hopefully this fixes it!

Reply:
We'll see. I'm running tests locally with `-j1` so I can find the hanging test, if there is still one after this.